### ISSUE TITLE
Conky integration

### ISSUE THREAD

**Issue by dragonator4** [RBlafford/levd#4]
Hi, any idea how I can get this integrated with Conky? I can simply read temperatures, fan and pump speeds from a file. Like [jaksi/leviathan](https://github.com/jaksi/leviathan) writes to files in `/sys/bus/drivers/usb/`, in which case, I just `cat` it from there in Conky.

**Comment by RBlafford** [comment]
If you need to simply read/write from a file maybe its better to use jaski's implementation? He wrote a driver, where this utility is a daemon.

**Comment by dragonator4** [comment]
It is true that Jaksi wrote a driver, but I find that a daemon has better UI and does not break with every kernel update. I am on Arch, so kernel gets updated quite frequently, and I found myself rebuilding the driver each time. A daemon like service such as this can be started and stopped as I need, and since there are no other programs that interface with it, I don't really see a reason to have the driver. Maybe you all could integrate the driver into the daemon..? I would do it, but doing so, and having the driver built and installed with every kernel update is beyond my programming skills in C.

Moreover, this project and the forks have the fan/pump speed regulation built in. Just made sense to use this instead.

I ended up using Jrudess' fork. I modified it to write the temperatures and speeds to a file that I can read with Conky. So that solves this. I can place a PR for my changes (which is just comprised of writing speeds and temps to a file) if you want. I am afraid that I can only do that around March, as I will be travelling away from my desktop till then.

**Comment by RBlafford** [comment]
Yeah sure put up a PR and i'll take a look at it, glad to hear your points, i'd be interested in using Conky myself, never heard of it

**Comment by RBlafford** [comment]
@dragonator4 I also just installed Conky now, would you mind sharing your conky configuration file for levd? Thanks!
